src/rules/prefer_primordials.rs

```diff
@@ -671,69 +678,6 @@ const noop = Function.prototype;
           hint: PreferPrimordialsHint::GlobalIntrinsic,
         },
       ],
-      r#"/aaaa/u.dotAll;"#: [
-        {
-          col: 0,
-          message: PreferPrimordialsMessage::GlobalIntrinsic,
-          hint: PreferPrimordialsHint::GlobalIntrinsic,
-        },
-      ],
-      r#"/aaaa/u.flags;"#: [
-        {
-          col: 0,
-          message: PreferPrimordialsMessage::GlobalIntrinsic,
-          hint: PreferPrimordialsHint::GlobalIntrinsic,
-        },
-      ],
```

Comment:
Why not add some test cases to "valid" test function so that we can make sure these cases don't trigger this lint error anymore?

Reply:
Sounds good. It would be too redundant to leave all of it, so I will leave some of it.